### PR TITLE
Mailer

### PR DESCRIPTION
**Pull request by lequytra**

### CODE REVIEW

File: app/services/notifier/email_notifier.rb [hunk 1]
```diff
@@ -1,11 +1,69 @@
...
 module Notifier
-  class EmailNotifier < Base
+  class EmailNotifier < Notifier::Base
+    def initialize(real_messages)
+      super real_messages
+      if real_messages
+        @mg_client = Mailgun::Client.new ENV['MAILGUN_API_KEY']
+      end
+      @logger = Rails.logger
+      @from_email = "grinnell-rideboard@#{ENV['MAILGUN_DOMAIN']}"
```
Comment (AlexanderOtavka) [inline app/services/notifier/email_notifier.rb]: Could we make this notifications@...?  I think being more specific is good

File: config/environments/development.rb [hunk 2]
```diff
@@ -33,11 +42,11 @@
...
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
 
   config.action_mailer.perform_caching = false
-
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  #
+  #config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (AlexanderOtavka) [inline config/environments/development.rb]: Why does this get commented out?

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: "ride-board@#{EVN['MAILGUN_DOMAIN']}"
```
Comment (AlexanderOtavka) [inline app/mailers/application_mailer.rb]: Yeah, this should be ENV not EVN

File: config/environments/development.rb (hunk 2 above)
Comment (lequytra) [reply config/environments/development.rb]: I set it to the mg.rideboard.app instead

**Pull request closed without merging**